services/121-service/test/e2e/.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules/
```

Comment:
Yes so initial idea from my side was to keep it separate but then idea of keeping all tests together won. And I kept them literally together like happy family under one directory .

"We leave api within ROOT/services/121-service/test and we move e2e to ROOT/e2e on the top-level (this would be my preference)"

This is good idea and I could separate it with next PR/s

Reply:
I think it would be easier to separate it now, so we don't touch the api/tests folder at all and have a cleaner history. I'm happy to help with this step if it is a git hassle.

Reply:
No worries working on it with Ashish with a bit of workaround.

Reply:
I just moved the full e2e directory to the root level to keep it separate!
@Piotrk39 Could you please test it now that its working accordingly?